app/Http/Controllers/ArticleController.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Article;
+use Illuminate\Http\Request;
+
+class ArticleController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // ④DBから記事情報を取得して変数に代入。
+        // $articles = Article::all();
+
+        // 更新した順に並び替え
+        $articles = Article::orderby('created_at', 'desc')->get();
+
+        //③記事一覧画面を表示。articleというリンク先（名前？）についてはlayouts/dashboardというものをviewさせるという意味になる！compactには上記の変数を入れており、DBからのデータを反映させている。
+        return view('layouts/dashboard', compact('articles'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+        // 記事投稿画面を表示
+        return view('layouts/create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
```

Comment:
```suggestion
     * @return \Illuminate\Http\RedirectResponse
```
こっちが正で、useをするとこう書き換えになる

 ```php
 use Illuminate\Http\RedirectResponse
 // (略)
 @return RedirectResponse
 ```

Reply:
<img width="1380" alt="スクリーンショット 2022-12-05 23 32 40" src="https://user-images.githubusercontent.com/91363928/205662764-3ef2b7c9-be34-41a7-a0b6-bb9f064bb2c5.png">
画像のように修正した。これでよいか。

Reply:
@return \Illuminate\Http\RedirectResponseに修正済。